Library/Formula/sslyze.rb

```diff
@@ -0,0 +1,52 @@
+class Sslyze < Formula
+  homepage "https://github.com/nabla-c0d3/sslyze"
+  url "https://github.com/nabla-c0d3/sslyze/archive/release-0.11.tar.gz"
+  sha256 "d0adf9be09d5b27803a923dfabd459c84a2eddb457dac2418f1bf074153f8f93"
+  version "0.11.0"
+
+  depends_on :arch => :x86_64
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "nassl" do
+    url "https://github.com/nabla-c0d3/nassl/archive/v0.11.tar.gz"
+    sha256 "83fe1623ad3e67ba01a3e692211e9fde15c6388f5f3d92bd5c0423d4e9e79391"
+  end
+
+  resource "openssl" do
+    url "https://www.openssl.org/source/openssl-1.0.2a.tar.gz"
+    sha256 "15b6393c20030aab02c8e2fe0243cb1d1d18062f6c095d67bca91871dc7f324a"
+  end
+
+  resource "zlib" do
+    url "http://zlib.net/zlib-1.2.8.tar.gz"
+    sha256 "36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d"
+  end
+
+  def install
+    ENV.deparallelize
+
+    resource("openssl").stage do
+      (buildpath/"nassl/openssl-1.0.2a").install Dir["*"]
+    end
+
+    resource("zlib").stage do
+      (buildpath/"nassl/zlib-1.2.8").install Dir["*"]
+    end
+
+    resource("nassl").stage do
+      (buildpath/"nassl").install Dir["*"]
+    end
+
+    cd "nassl" do
+      system "python", "buildAll_unix.py"
+      libexec.install "test/nassl"
+    end
+
+    libexec.install %w[plugins utils sslyze.py xml_out.xsd]
+    bin.install_symlink libexec/"sslyze.py" => "sslyze"
+  end
+
+  test do
+    system "#{bin}/sslyze", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!